client/src/models/history/CreatePart.ts

```diff
@@ -17,18 +19,31 @@ export const create: Create = async (apiType, params) => {
   return part
 }
 
-export const getBufferGeom = async (productId: number, api: ApiHistory) => {
+// export const getBufferGeom = async (productId: number, api: ApiHistory) => {
+//   if (!api) return
+//   const geoms = await api.createBufferGeometry(productId)
+//   return geoms.map(
+//     geom =>
+//       new THREE.Mesh(
+//         geom,
+//         new THREE.MeshStandardMaterial({ color: new THREE.Color('rgb(52, 89, 87)') }),
+//       ),
+//   )
+// }
```

Comment:
Does it make sense to keep this code?

Reply:
Doesn't make sense. You're right. I removed it. See [64653ef](https://github.com/awv-informatik/buerli-examples/pull/11/commits/64653efebb741eea6b47350fbc24f16b2a8bd2da)